src/app/builds/build-details/build-details.component.ts

```diff
@@ -65,4 +67,19 @@ export class BuildDetailsComponent implements OnInit, OnDestroy {
   ngOnDestroy() {
     this.source?.removeEventListener('message', this.listener);
   }
+
+  private stayScrolledToBottom(): void {
+    if (this.isScrolledToBottom()) {
+      this.scrollToBottom();
+    }
+  }
```

Comment:
Any chance this makes it impossible to scroll up?

Reply:
If the logs are spamming in then yes there's a chance. But I've tested and as soon as you're a little bit scrolled up then it won't drag you down. The -2 makes it a 2 px margin, which you should be able to scroll quite quickly. Especially since most scroll wheels scroll big chunks at a time.